android/src/main/java/com/twiliorn/library/CustomTwilioVideoViewManager.java

```diff
@@ -179,6 +183,10 @@ public Map getExportedCustomDirectEventTypeConstants() {
                 ON_DOMINANT_SPEAKER_CHANGED, MapBuilder.of("registrationName", ON_DOMINANT_SPEAKER_CHANGED)
         ));
 
+        map.putAll(MapBuilder.of(
+                ON_LOCAL_PARTICIPANT_H246_SUPPORTED, MapBuilder.of("registrationName", ON_LOCAL_PARTICIPANT_H246_SUPPORTED)
```

Comment:
H246 ==> H264

Reply:
Any reason to create a new `map.putAll(MapBuilder.of(` as oppose to add this line after l183?

Reply:
Not opposed to it, I just saw similar that other people added it seperately